keyboards/ut47/keymaps/nordic/config.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2018 Carlos Filoteo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```

Comment:
Could this line and everything after it be replaced with `#pragma once`?

Reply:
Not sure I understand this one? I just copy/pasted the config.h file from the default UT47 keymap.

Reply:
Basically, replace this: 
```h
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "config_common.h"

// place overrides here

#endif
```

with just: 
```h
#pragma once 

```

Reply:
Alright that's sorted now too!